docs/paper/paper.md

```diff
@@ -87,8 +87,10 @@ The implementation for the plane-cylinder geometry is based on [@emig_prl_2006].
 
 ``CaPS`` has been used to analyze negative Casimir entropies [@ingold_pre_2015;
 @umrath_pre_2015], and to study corrections to the widely used proximity force
-approximation for experimentally relevant parameters [@hartmann_prl_2017]. The
-package is released under the GPLv2 license.
+approximation for experimentally relevant parameters [@hartmann_prl_2017].  In
+addition, data generated using ``CaPS`` was used to analyze the experiments in
```

Comment:
"was" -> "were"

Reply:
I have fixed the grammar mistake (a8699b20d8a263169a4d03d77dec22badd8c5e4c).